jms/src/main/scala/akka/stream/alpakka/jms/JmsConsumerSettings.scala

```diff
@@ -24,12 +24,17 @@ final class JmsConsumerSettings private (
     val acknowledgeMode: Option[AcknowledgeMode],
     val ackTimeout: scala.concurrent.duration.Duration,
     val failStreamOnAckTimeout: Boolean,
-    val connectionStatusSubscriptionTimeout: scala.concurrent.duration.FiniteDuration
+    val connectionStatusSubscriptionTimeout: scala.concurrent.duration.FiniteDuration,
+    val consumer: Option[javax.jms.MessageConsumer]
```

Comment:
I was not able to consume a temporary queue destination with connections created by the connection factory.  To create a `JmsConsumer` that could stream the "replies", I had to use the same consumer from the session that was used to create the temporary queue.